FinSearchUnified/Resources/views/frontend/listing/filter/_includes/filter-color-selection.tpl

```diff
@@ -13,18 +13,19 @@
                value="{$option->getId()|escape:'htmlall'}" style="background-color: #fff !important; content: ''"
                {if $option->isActive()}checked="checked" {/if}/>
 
-                                                            <span class="input--state {$inputType}--state">&nbsp;</span>
-                                                        </span>
+        <span class="input--state {$inputType}--state">&nbsp;</span>
+    </span>
 {/block}
 
 {block name="frontend_listing_filter_facet_multi_selection_label"}
     <label class="filter-panel--label"
            for="__{$facet->getFieldName()|escape:'htmlall'}__{$option->getId()|escape:'htmlall'}">
 
-            {if $option->getColorcode()}
-                {$mediaFile = $option->getColorcode()}
-            {/if}
-            <div style="width:30px; height: 30px; margin: 2px; background: {$mediaFile};"></div>
-
+        {if $option->getColorcode()}
+            {$mediaFile = $option->getColorcode()}
+            <div style="width:30px; height: 30px; margin: 2px; border: 1px solid #dbd7e5; background: {$mediaFile};" title="{$option->getId()|escape:'htmlall'}"></div>
+        {else}
+            <img style="width:30px; height: 30px; border: 1px solid #dbd7e5; margin: 2px;" class="filter-panel--media-image" title="{$option->getId()|escape:'htmlall'}" src="/custom/plugins/FinSearchUnified/Resources/views/frontend/_public/src/img/no-picture.png" alt="{$option->getId()|escape:'htmlall'}">
```

Comment:
Create a new LESS file for these styles. We already have some for the promotion for example.